Mixpanel/AutomaticProperties.swift

```diff
@@ -47,7 +65,9 @@ class AutomaticProperties {
             p["$ios_app_release"] = infoDict["CFBundleShortVersionString"]
         }
         p["$ios_device_model"]  = AutomaticProperties.deviceModel()
+        #if !MAC_OS
         p["$ios_version"]       = UIDevice.current.systemVersion
```

Comment:
should we put the macOS version in this too? since we're putting other random properties in $ios_blah